### DISCUSSION TITLE
Very slow download

### DISCUSSION THREAD

**Discussion by chansters** [Tzahi12345/YoutubeDL-Material#505] [Q&A] [answered]
Download speeds are really slow and it seems like Youtube-DL is being throttled by Youtube.  This [discussion](https://www.reddit.com/r/youtubedl/comments/qfbyal/read_slow_youtube_downloads/) recommends switching from Youtube-DL to [yt-dlp](https://github.com/yt-dlp/yt-dlp/). 

Is there anyway to make this switch while retaining the excellent GUI?

**Comment by GlassedSilver** [comment, marked as answer]
You switch the back-end engine from classic yt-dl to yt-dlp.
You do not switch away from YoutubeDL-Material.

1. Go to **Settings** > **Advanced** > **"Select a downloader"**
2. Pick **yt-dlp** in the dropdown list

**Reply by GlassedSilver** [reply]
I marked this as answer, but if this problem persists for you let me know. :)

One thing though: it might take a while for speeds to be up to snuff again, a restart of the container/machine/YTDL-M might help as well (whatever your setup is).

**Reply by chansters** [reply]
> I marked this as answer, but if this problem persists for you let me know. :)
> 
> One thing though: it might take a while for speeds to be up to snuff again, a restart of the container/machine/YTDL-M might help as well (whatever your setup is).

Thank you!

**Reply by itsmeduhdoi** [reply]
@GlassedSilver 

Which version do i need to be on to have yt_dlp be in the drop down list, currently on v4.2 and only have yt-dl and yt-dlc as options.

**Reply by GlassedSilver** [reply]
@itsmeduhdoi
Sorry for the late reply. :O 
You have to use the nightly builds. You‘re probably using the latest release build which is missing a lot of what‘s changed and improved.
